daml-foundations/daml-ghc/src/DA/Daml/GHC/Compiler/Convert.hs

```diff
@@ -1270,6 +1267,15 @@ mkPure env monad dict t x = do
           `ETyApp` t
           `ETmApp` x
 
+-- | Turn the structural record referenced by `x` into a tuple from `GHC.Tuple`.
```

Comment:
Would have been nice to get a little more details about the input structural record invariants in the comment. I assume it's `{_1 : a, _2: b, ...}`?

Reply:
I thought that would be obvious from the `[(FieldName, LF.Type)]` part of the type signature. I'll improve it in a followup PR.

Reply:
Maybe it is now I look closer... but I guess i still don't know what the invariants on the field names are.

Reply:
The fields of the input structural record are put into the tuple in the order they appear in the list. The tuple's fields are name `_1`, `_2`, ... as usual. I'll given an example in the code.